src/messages.js

```diff
@@ -562,36 +575,23 @@ function superBotMessageBlocks(inputs) {
         },
         {
             "type": "input",
-            "block_id": 'team_block',
+            "block_id": 'team_check_block',
             "label": {
                 "type": "plain_text",
                 "text": "Checked With Team?"
             },
             "element": {
                 "type": "plain_text_input",
-                "action_id": "team_input",
-                "initial_value": inputs?.team?.value ?? ""
-            }
-        },
-        {
-            "type": "input",
-            "block_id": 'action_block',
-            "label": {
-                "type": "plain_text",
-                "text": "Action Required"
-            },
-            "element": {
-                "type": "plain_text_input",
-                "action_id": "action_input",
-                "initial_value": inputs?.action?.value ?? ""
+                "action_id": "team_check_input",
+                "initial_value": inputs?.team_check?.value ?? ""
             }
         },
         {
             "type": "input",
             "block_id": 'user_block',
             "label": {
                 "type": "plain_text",
-                "text": "Ticker Raiser"
+                "text": "Ticket Raiser"
```

Comment:
why do we have this?

Reply:
We need to provide a user to the workflow so it knows who to put under the 'Reporter' field in the actual ticket. The old method just gets this from the body of the event, but I don't think you can do that with a workflow, hence why we're passing it in here

Reply:
needs checking.

Reply:
(discussed on slack, this works but is overly complex and the form should be moved completely to code, with pre-request bits staying in the workflow)